### PR TITLE
Subtitulamos.tv

### PR DESCRIPTION
**Pull request by gmerciel**
Add [Subtitulamos.tv](https://www.subtitulamos.tv/) spanish subtitles provider.

### CODE REVIEW

**Comment by morpheus65535** [comment]
#525

**Comment by gmerciel** [comment]
I'm not requesting a feature, but providing a pull request to be merged. This is the [Feature Request](https://bazarr.featureupvote.com/suggestions/54298/provider-request-wwwsubtitulamostv) for reference.

**Comment by morpheus65535** [comment]
Lol OMG sorry I’ll look into it later, thanks!!

File: libs/subliminal_patch/providers/subtitulamostv.py [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+from guessit import guessit
+from requests import Session
+from subzero.language import Language
+
+from subliminal import Movie, Episode, ProviderError, __short_version__
+from subliminal.exceptions import AuthenticationError, ConfigurationError, DownloadLimitExceeded, ProviderError
+from subliminal_patch.subtitle import Subtitle, guess_matches
+from subliminal.subtitle import fix_line_ending, SUBTITLE_EXTENSIONS
+from subliminal_patch.providers import Provider
+from subzero.language import Language
```
Comment (morpheus65535) [inline libs/subliminal_patch/providers/subtitulamostv.py]: Duplicate of line 11.

Comment (morpheus65535) [reply libs/subliminal_patch/providers/subtitulamostv.py]: Are you still there?

Comment (gmerciel) [reply libs/subliminal_patch/providers/subtitulamostv.py]: Fixed on last commit

**Comment by gmerciel** [comment]
Sorry for the delay, I wanted to fix an issue I was having with the encoding: the encoding reported on the HTTP header is UTF-8 but the files are actually ISO-8859-1 encoded.

**Comment by morpheus65535** [comment]
Make your PR to dev branch please.

**Comment by gmerciel** [comment]
#653

**Pull request closed without merging**